### DISCUSSION TITLE
Little problem in navigation links on "https://allurereport.org/docs/integrations-github/" pages

### DISCUSSION THREAD

**Discussion by ghost** [allure-framework/allure2#2803] [Bug Reports] [answered]
On this webpage of Allure integrations the navigation links given on the page in the center are somehow not navigating accordingly. 
![image](https://github.com/user-attachments/assets/b0d7ea60-cefd-4451-9d50-c82c6815feeb)

**Comment by baev** [comment, marked as answer]
@GuravBhargav09, thanks for the report! The issue is now fixed.
